stage1/rootfs/usr_from_src/Makefile

```diff
@@ -31,10 +31,12 @@ systemd.done: Makefile systemd.src
 
 # TODO(vc): it may make more sense to have the systemd source be a git submodule?
 systemd.src: Makefile patches/*
+	if [ -z "$(RKT_STAGE1_SYSTEMD_SRC)" ]; then echo "Error: RKT_STAGE1_SYSTEMD_SRC undefined"; exit 1; fi
+	if [ -z "$(RKT_STAGE1_SYSTEMD_VER)" ]; then echo "Error: RKT_STAGE1_SYSTEMD_VER undefined"; exit 1; fi
 	{ [ ! -e systemd ] || rm -Rf systemd; }
 	mkdir systemd
-	git clone --branch v215 --depth 1 git://anongit.freedesktop.org/systemd/systemd
-	cd systemd && for p in ../patches/*; do patch -p1 <"$$p"; done
+	git clone --branch $(RKT_STAGE1_SYSTEMD_VER) --depth 1 $(RKT_STAGE1_SYSTEMD_SRC)
+	if [ -d patches/$(RKT_STAGE1_SYSTEMD_VER) ]; then cd systemd && for p in ../patches/$(RKT_STAGE1_SYSTEMD_VER)/*.patch; do patch -p1 <"$$p"; done; fi
```

Comment:
The problems I see with this loop are:
1. If patches directory is empty then the for loop will be iterated once with `patch -p1 <"../patches/<VERSION>/*.patch"`. Since we assume we are running bash, we could run `shopt -s nullglob` before entering the loop.
2. Given several patches to apply, if all but last fail to be applied this loop won't terminate `make`'s operation.

IMO, it should be something like this:
```bash
if [ -d patches/$(RKT_STAGE1_SYSTEMD_VER) ]
then
    set -e
    cd systemd
    shopt -s nullglob
    for p in ../patches/$(RKT_STAGE1_SYSTEMD_VER)/*.patch
    do
        if patch -p1 <"$$p"
        then
            :
        else
            exit 1
        fi
    done
fi
```